hadoop-hdds/docs/content/feature/PrefixFSO.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Prefix based FileSystem Optimization"
+title: "Prefix based File System Optimization"
```

Comment:
Howe about including the acronym **FSO** in the heading ?

`Prefix based File System Optimization (FSO).`